src/main/kotlin/com/example/venda_ingressos/controller/ClientController.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.venda_ingressos.controller
+
+import com.example.venda_ingressos.entities.Client
+import com.example.venda_ingressos.service.ClientService
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/client", consumes = ["application/json"])
```

Comment:
O caminho do controller deve estar no plural
```suggestion
@RequestMapping("/clientes", consumes = ["application/json"])
```